bhs_api/fsearch.py

```diff
@@ -29,13 +29,14 @@
                 }
 
 def _generate_year_range(year, fudge_factor=0):
-    maximum = int(str(year + fudge_factor) + '9999')
-    minimum = int(str(year - fudge_factor) + '0000')
+    maximum = int(str(year + fudge_factor))
+    minimum = int(str(year - fudge_factor))
```

Comment:
not sure why the additional numbers were added, currently we store the year as-is, as a number in mongo